modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/FDSPropsTransformer/cell_renderers/SpaceRenderer.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import Sticker from '@clayui/sticker';
+import React from 'react';
+
+const SpaceRenderer = ({value}: {value: string}) => {
+	const isStructure = false;
+
+	return isStructure ? (
+		<span className="badge badge-pill badge-secondary">
+			<span className="badge-item badge-item-expand">
+				{Liferay.Language.get('all-spaces')}
+			</span>
+		</span>
+	) : (
+		<span className="align-items-center d-flex">
+			<Sticker
```

Comment:
Here I would suggest to use the new SpaceSticker (it was merged [here](https://github.com/brianchandotcom/liferay-portal/pull/159652)). Each space would have its own color in the future, so that component will handle the stickers. 
The only change needed would be to add a size property to that class, I was not aware of that need.

Reply:
Thanks for making that component! Will apply it for the cell renderer then